### DISCUSSION TITLE
E94 – Найцікавіші новини місяця: червень 2024пше

### DISCUSSION THREAD

**Discussion by malor** [shopokodu/community#68] [Теми для подкасту]
* https://ladybird.org/
* GenAI тепер в Firefox https://blog.mozilla.org/en/products/firefox/firefox-news/ai-services-on-firefox/
* https://medium.com/@austin-starks/i-spent-18-months-rebuilding-my-algorithmic-trading-in-rust-im-filled-with-regret-d300dcc147e0
* https://www.qualys.com/2024/07/01/cve-2024-6387/regresshion.txt

---

* https://robertheaton.com/pyskywifi/
* https://www.bleepingcomputer.com/news/security/dev-rejects-cve-severity-makes-his-github-repo-read-only/

**Comment by nextgeniuspro** [comment]
Є декілька цікавих пейперів з використання гаусіан для [рендерінга сцени](https://gaussiantracer.github.io/) та для [реконструкції сцени](https://wonderworld-2024.github.io/) за заданим імаджем. Було б класно почути експертів по цим питанням, якщо такі є

**Reply by ikalnytskyi** [reply]
На жаль, я в цьому не розбираюсь, а знайомий експерт морозиться :(
